src/runtime_src/core/include/experimental/xrt_xclbin.h

```diff
@@ -19,11 +19,202 @@
 #define _XRT_XCLBIN_H_
 
 #include "xrt.h"
+#include "experimental/xrt_uuid.h"
 
 #ifdef __cplusplus
+# include <memory>
+# include <vector>
+# include <string>
+#endif
+
+/**
+ * typedef xrtXclbinHandle - opaque xclbin handle
+ */
+typedef void* xrtXclbinHandle;
+
+#ifdef __cplusplus
+namespace xrt {
+
+class xclbin_impl;
+class xclbin
+{
+public:
+  
+  /**
+   * xclbin() - Copy ctor
+   */
+  xclbin(const xclbin& rhs)
```

Comment:
Why all these explicit constructors?
At least just default them.

Reply:
Yes. Start just with a single
```c++
xclbin() = default;
```

Reply:
Leaving this the same to conform to Soren's xrt::device class.